hue/README.md

```diff
@@ -0,0 +1,47 @@
+# Cloudera Hue
```

Comment:
It seems like this does not work without thrifserver, right? If so, can you mention it in the readme and maybe even as a comment in KFDef?

Reply:
It can work without thriftserver, but you'd need to connect it to something else (yarn, Hive directly, etc) via modifying the `hue.ini`. Or, you can use it as an S3 explorer. Which is a standalone functionality. In our scenario and set up it requires Thrift server to be available. I'll explain it in the readme. :+1:

Reply:
I've added a paragraph in there. WDYT? :slightly_smiling_face:

Reply:
This clarification looks good to me.